src/gd_kotlin.cpp

```diff
@@ -74,16 +76,16 @@ void unload_classes_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_classes)
 }
 
 void register_engine_types_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_engine_types, jobjectArray p_singleton_names,
-        jobjectArray p_method_names, jobjectArray p_types_of_methods) {
-    print_verbose("Starting to register managed engine types...");
+                                jobjectArray p_method_names, jobjectArray p_types_of_methods) {
+    LOG_VERBOSE("Starting to register managed engine types...")
```

Comment:
Maybe we can add verbose logs only for DEBUG builds, so that we don't get extra calls in release ?

Reply:
done